docs/source/common/trainer.rst

```diff
@@ -182,6 +182,8 @@ and set ``deterministic`` flag in ``Trainer``.
 
 Example::
 
+.. code-block:: python
+
```

Comment:
you cannot put this inside inside the example block, or it will become code. 
please check the output:

![image](https://user-images.githubusercontent.com/5495193/125582603-de45590d-77e7-412c-98bd-cf8222348107.png)

you can generate the docs like so:
- pip install -r requirements/docs.txt
- cd docs
- make html 
and then browse to docs/build and open the index.html file for example. then navigate to trainer.

Reply:
If you need help, let me know. 
Happy to assist @aslisabanci

Reply:
Ahh you're right! I could've even caught this from Github's preview - but of course I'll make the docs with your instructions and double check them this way from now on. 
For a quick fix I just removed the code-block sections from under Examples and sent a new commit, but for my next PRs I'll make sure to do this beforehand. This one was in rush to be the 500th contributor :)

If I need any assistance making the docs I'll reach out to you too, thanks a bunch for your support Adrian!